scripts/open-coredump.sh

```diff
@@ -162,10 +162,13 @@ do
             SCYLLA_GDB_PY_SOURCE=$2
             shift 2
             ;;
-        *)
+        "--*")
```

Comment:
Just for completeness' sake, this should be `"--*"|"-*"`, so it also matches uknown short-form options.